### DISCUSSION TITLE
want to unpack mini_obb.pak in BGMI ( PUBG Indian v4ersion)

### DISCUSSION THREAD

**Discussion by Calamivathan** [halloweeks/pubg-unpacker#6] [General]
hey just saw your github and Unknowncheats profile can you help me in unpacking BGMI( pubg Indian version) Pak files i am unable to do it and stuck at various thing looking out for your help if u could teach me how to do it
would be a great help
i want to learn to make my own mods in the obb 
pls help

**Comment by halloweeks** [comment]
Decryption key required!
